### ISSUE TITLE
Segfault on malware sample

### ISSUE THREAD

**Issue by DigiAngel** [radare/radare2#6391]
Sample:
~~~
https://www.virustotal.com/en/file/b8063f1323a4ae8846163cc6e84a3b8a80463b25b9ff35d70a1c497509d48539/analysis/
~~~

it's pretty nasty...anyway backtrace here:
~~~
[Fast goto call/jmp]> a
Program received signal SIGSEGV, Segmentation fault.
__strcmp_ssse3 () at ../sysdeps/x86_64/multiarch/../strcmp.S:173
173     ../sysdeps/x86_64/multiarch/../strcmp.S: No such file or directory.
(gdb) backtrace
#0  __strcmp_ssse3 () at ../sysdeps/x86_64/multiarch/../strcmp.S:173
#1  0x00007ffff7973386 in ds_instruction_mov_lea (ds=0x5555557d4a50, idx=13) at disasm.c:1841
#2  0x00007ffff7977123 in r_core_print_disasm (p=0x5555557c00e0, core=0x55555575bb20 <r>, addr=4464199,
    buf=0x555558964380 "f\207\376T\017\266\361\211\326f\017\317\375\215<\245\231\332\331\367\351-\r", len=25, l=25, invbreak=0, cbytes=1) at disasm.c:3116
#3  0x00007ffff79084dc in cmd_print (data=0x55555575bb20 <r>, input=0x555556d885c1 "D 25") at cmd_print.c:2804
#4  0x00007ffff7954f60 in r_cmd_call (cmd=0x5555558104d0, input=0x555556d885c0 "pD 25") at cmd_api.c:210
#5  0x00007ffff791fce9 in r_core_cmd_subst_i (core=0x55555575bb20 <r>, cmd=0x555556d885c0 "pD 25", colon=0x0) at cmd.c:1822
#6  0x00007ffff791dc0d in r_core_cmd_subst (core=0x55555575bb20 <r>, cmd=0x555556d885c0 "pD 25") at cmd.c:1255
#7  0x00007ffff7921894 in r_core_cmd (core=0x55555575bb20 <r>, cstr=0x7fffffffab10 "pD 25 @ 0x00441e47", log=0) at cmd.c:2337
#8  0x00007ffff792213f in r_core_cmd_str (core=0x55555575bb20 <r>, cmd=0x7fffffffab10 "pD 25 @ 0x00441e47") at cmd.c:2524
#9  0x00007ffff79220fa in r_core_cmd_strf (core=0x55555575bb20 <r>, fmt=0x7ffff79ad5d7 "%s %d @ 0x%08llx") at cmd.c:2514
#10 0x00007ffff793f2d9 in get_body (core=0x55555575bb20 <r>, addr=4464199, size=25, opts=0) at graph.c:1577
#11 0x00007ffff793f568 in get_bbnodes (g=0x5555570c88a0, core=0x55555575bb20 <r>, fcn=0x555556c60830) at graph.c:1630
#12 0x00007ffff793fa81 in reload_nodes (g=0x5555570c88a0, core=0x55555575bb20 <r>, fcn=0x555556c60830) at graph.c:1737
#13 0x00007ffff7940b5e in agraph_reload_nodes (g=0x5555570c88a0, core=0x55555575bb20 <r>, fcn=0x555556c60830) at graph.c:2056
#14 0x00007ffff7940efa in check_changes (g=0x5555570c88a0, is_interactive=1, core=0x55555575bb20 <r>, fcn=0x555556c60830) at graph.c:2107
#15 0x00007ffff79411cf in agraph_print (g=0x5555570c88a0, is_interactive=1, core=0x55555575bb20 <r>, fcn=0x555556c60830) at graph.c:2155
#16 0x00007ffff79415ba in agraph_refresh (grd=0x55555a3a8130) at graph.c:2236
#17 0x00007ffff7942c74 in r_core_visual_graph (core=0x55555575bb20 <r>, g=0x5555570c88a0, _fcn=0x0, is_interactive=1) at graph.c:2619
#18 0x00007ffff7932cdb in r_core_visual_cmd (core=0x55555575bb20 <r>, ch=86) at visual.c:1534
#19 0x00007ffff7935de0 in r_core_visual (core=0x55555575bb20 <r>, input=0x55555a83fbc1 "") at visual.c:2348
#20 0x00007ffff791ca91 in cmd_visual (data=0x55555575bb20 <r>, input=0x55555a83fbc1 "") at cmd.c:881
#21 0x00007ffff7954f60 in r_cmd_call (cmd=0x5555558104d0, input=0x55555a83fbc0 "v") at cmd_api.c:210
#22 0x00007ffff791fed3 in r_core_cmd_subst_i (core=0x55555575bb20 <r>, cmd=0x55555a83fbc0 "v", colon=0x0) at cmd.c:1863
#23 0x00007ffff791dc0d in r_core_cmd_subst (core=0x55555575bb20 <r>, cmd=0x55555a83fbc0 "v") at cmd.c:1255
#24 0x00007ffff7921894 in r_core_cmd (core=0x55555575bb20 <r>, cstr=0x5555558c8380 "v", log=1) at cmd.c:2337
#25 0x00007ffff78bf624 in r_core_prompt_exec (r=0x55555575bb20 <r>) at core.c:1651
#26 0x00005555555598d6 in main (argc=2, argv=0x7fffffffe5a8, envp=0x7fffffffe5c0) at radare2.c:969
(gdb)
~~~

**Comment by Maijin** [comment]
What is this 

```
[Fast goto call/jmp]> a
```

That's not radare2.

Also

Ensure you are using radare2 from git, if you're unsure paste output of `r2 -v` here.
To install radare2 from git, first uninstall your version of radare2 and clean your distro. Then use `git clone https://github.com/radare/radare2 && cd radare2 && ./sys/install.sh`, verify your version and check if there is no error using `r2 -v`.

**Comment by Maijin** [comment]
And please details where you are doing this `[Fast goto call/jmp]> a`, Use an asciinema

**Comment by DigiAngel** [comment]
Version here:
~~~
radare2 0.10.6-git 12316 @ linux-x86-64 git.0.10.5-182-ga458a36
commit: a458a368145cf26ad6bb4e6cab28ea986b463c0a build: 2016-09-07
~~~

That call/jmp is per the help in Visual mode: 
~~~
g([A-Za-z}*) - follow jmp/call identified by shortcut
~~~
I'll see if I can get asciinema from a linux box later today.

**Comment by radare** [comment]
0.10.6 is pretty old. Use git to confirm the bug

**Comment by DigiAngel** [comment]
Well that was interesting.....looks like when I initially installed I did a git clone then ./configure make sudo make install.  Since then I've been doing sudo sys/install.sh, which apparently installs somewhere else besides /usr/local/.  So now here's what I got:
~~~
WARNING: r2 library versions mismatch! See r2 -V
radare2 1.2.0-git 13374 @ linux-x86-64 git.1.1.0-83-g7b39138ce
commit: 7b39138ce79fca762436554c090bc51caf08b1af build: 2016-12-28
[16:40:13 @cuckoo:~$] r2 -V
1.1.0-83-g7b39138ce  r2
0.10.5-182-ga458a36  r_anal
0.10.5-182-ga458a36  r_lib
0.10.5-182-ga458a36  r_egg
0.10.5-182-ga458a36  r_asm
0.10.5-182-ga458a36  r_bin
0.10.5-182-ga458a36  r_cons
1.1.0-83-g7b39138ce  r_flag
0.10.5-182-ga458a36  r_core
0.10.5-182-ga458a36  r_crypto
0.10.5-182-ga458a36  r_bp
0.10.5-182-ga458a36  r_debug
0.10.5-182-ga458a36  r_hash
0.10.5-182-ga458a36  r_fs
0.10.5-182-ga458a36  r_io
0.10.5-182-ga458a36  r_magic
0.10.5-182-ga458a36  r_parse
0.10.5-182-ga458a36  r_reg
0.10.5-182-ga458a36  r_sign
0.10.5-182-ga458a36  r_search
0.10.5-182-ga458a36  r_syscall
0.10.5-182-ga458a36  r_util
~~~
with the same segfault.  I thought I'd cleaned out any bits of 0.10.5, but it looks like not..where else can I check?  Thank you.
